thanks @vidigoat 

we get a CI failure:

```
FAILED tests/expr_and_series/dt/ordinal_day_test.py::test_ordinal_day - pandas.errors.Pandas4Warning: DatetimeIndex.dayofyear is deprecated and will be removed in a future version. Use DatetimeIndex.day_of_year instead.
Falsifying example: test_ordinal_day(
    dates=datetime.datetime(1980, 1, 1, 0, 0),  # or any other generated value
)
```

could you fix up the code accordingly please?

--- a/src/narwhals/_pandas_like/series_dt.py
+++ b/src/narwhals/_pandas_like/series_dt.py
@@ -88,7 +88,11 @@ class PandasLikeSeriesDateTimeNamespace(
         return self.microsecond() * 1_000 + self.native.dt.nanosecond
 
     def ordinal_day(self) -> PandasLikeSeries:
-        return self.with_native(self.native.dt.dayofyear)
+        # Use ``day_of_year`` rather than the ``dayofyear`` alias: the two are
+        # equivalent on all supported pandas versions (the alias exists since
+        # 1.1.0), but ``dayofyear`` is deprecated on pandas' main branch, which
+        # turns the nightly CI (warnings-as-errors) red.
+        return self.with_native(self.native.dt.day_of_year)
 
     def weekday(self) -> PandasLikeSeries:
         # Pandas is 0-6 while Polars is 1-7

--- a/src/narwhals/_pandas_like/series_dt.py
+++ b/src/narwhals/_pandas_like/series_dt.py
@@ -88,10 +88,6 @@ class PandasLikeSeriesDateTimeNamespace(
         return self.microsecond() * 1_000 + self.native.dt.nanosecond
 
     def ordinal_day(self) -> PandasLikeSeries:
-        # Use ``day_of_year`` rather than the ``dayofyear`` alias: the two are
-        # equivalent on all supported pandas versions (the alias exists since
-        # 1.1.0), but ``dayofyear`` is deprecated on pandas' main branch, which
-        # turns the nightly CI (warnings-as-errors) red.
         return self.with_native(self.native.dt.day_of_year)
 
     def weekday(self) -> PandasLikeSeries:

--- a/tests/expr_and_series/dt/datetime_attributes_test.py
+++ b/tests/expr_and_series/dt/datetime_attributes_test.py
@@ -93,8 +93,6 @@ def test_datetime_attributes_series(
 
 
 def test_ordinal_day_preserves_null(constructor: Constructor) -> None:
-    # ordinal_day previously dropped nulls on the pandas-like backend (it
-    # returned 1 for NaT), unlike every other datetime attribute.
     data_with_null = {"a": [datetime(2021, 3, 1), None]}
     df = nw.from_native(constructor(data_with_null))
     result = df.select(nw.col("a").dt.ordinal_day())

